test/definers/readme.test.ts

```diff
@@ -1,15 +1,10 @@
 import {describe, expect, test} from 'vitest'
-import {defineBlueprint, defineDocumentFunction, defineResource} from '../../src/index.js'
+import {defineBlueprint, defineDocumentFunction, defineDocumentWebhook} from '../../src/index.js'
 
 describe('README example', () => {
   test('should be valid', () => {
     const readmeExample = defineBlueprint({
       resources: [
-        defineDocumentFunction({name: 'invalidate-cache', projection: '_id'}),
```

Comment:
Updates in this test are just updating the test to accurately reflect the README